net-analyzer/prelude-lml-rules/prelude-lml-rules-3.0.1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Prelude LML community ruleset"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="net-analyzer/prelude-lml"
+
+RDEPEND="${DEPEND}"
+
+src_install() {
+	install -d -m 0755 "${D}/etc/prelude-lml/ruleset"
```

Comment:
You should make use of the [install functions](https://devmanual.gentoo.org/function-reference/install-functions/index.html) provided by the package manager here. i.e. instinto, dodir, doins, etc.